tasks/push-rpm-data-to-pyxis/tests/test-push-rpm-data-to-pyxis-cyclonedx.yaml

```diff
@@ -60,7 +61,7 @@ spec:
         name: push-rpm-data-to-pyxis
       params:
         - name: pyxisJsonPath
-          value: pyxis_data.json
+          value: testdir/pyxis_data.json
```

Comment:
I'm fine with this, but note that there is really no special reason for this change. It's pretty common for our tests to use the top level of the workspace, whereas in the real pipeline, we always use a subdirectory.

Reply:
The Tekton task tests were failing without this change, because with the use of `dirname`, something like this happens:
Original code:
```
SBOM_PATH="$(dirname "$(params.pyxisJsonPath)")/downloaded-sboms" 
cd "$(workspaces.data.path)/${SBOM_PATH}"
```
At runtime:
```
SBOM_PATH=./downloaded-sboms
cd /workspace/data/./downloaded-sboms 
```
The second command results in a Not Found error

Reply:
I tried it myself and it worked just fine. You can add as many `./` as you like in a path and it will still work, no issue with that. This is what I did and then this test still passed: https://github.com/mmalina/release-service-catalog/commit/8cef806502f86a765d94d14ffe7ced4d72c5f76f

Reply:
Removed the `testdir` and got the tests to pass.

Reply:
OK, thanks.